nix/build/zcash-overlay.nix

```diff
@@ -15,7 +15,7 @@ let
   };
 
   relevant-packages =
-    "tezos-((node|client|codec|admin-client|signer)|((accuser|baker).*))";
+    "(octez-((node|client|codec|signer)|((accuser|baker).*)))|(tezos-admin-client)";
```

Comment:
oh what, is `tezos-admin-client` the only exception?

Reply:
yep

Reply:
something seems off to me, because [from the announcement](https://research-development.nomadic-labs.com/filename-renaming-octez-v15.html) I see that:
> `tezos-admin-client` becomes `octez-admin-client`.

just like every other binary and indeed AFAICT the binary _has_ been renamed.

Can you please check again that this isn't caused by some exception in our code?

Reply:
it appear to be exception in our code eventually